frontend-web/src/coffee-call/call-aggregation.ts

```diff
@@ -0,0 +1,71 @@
+import { delay, map, merge, Observable, of, scan, switchMap, tap } from "rxjs";
+import { ReceivedCallMessage } from "./protocol";
+
+export const enum CoffeeCallState {
+  inactive,
+  announced,
+  inProgress,
+  canceled,
+};
+
+export class CoffeeCall {
```

Comment:
We also have a component with that name, this is quite confusing… :-/

How about `CallStateMachine`?

Reply:
I know, this name-clash is bad and even worse is the import in the UI. I was rather inclined to rename the component, though? I would have cleaned up the directory structure in an upcoming PR anyhow.

Reply:
Fine for me.